core/route/allrouting.js

```diff
@@ -87,6 +87,18 @@ class Routing {
     }
 
     /**
+     * Match Method
+     * @param {String[]} methods - Array of methods to use for a route
+     * @param {String} path - Path for routing
+     * @param {String|Function} controllerAction
+     */
+    match(methods, path, controllerAction) {
+        methods.forEach((method) => {
+            this[method.toLowerCase()](path, controllerAction);
```

Comment:
I had thought about that but had overlooked it at the time due to us not having a consistent means for returning errors. Regardless we will need it in the end so I'll add that fix real quick.